FRE/fre_make.json

```diff
@@ -149,7 +149,11 @@
                 "makeOverrides": {
                     "description": "Overrides openmp target for MOM6",
                     "type": "string"
-                }
+                },
+                "additionalInstructions": {
+                    "description": "Additional instuctions to run after checkout",
+                    "type": "string"
+				}
```

Comment:
The indentation on the closing bracket looks a bit funky.

Reply:
Ooop, didn't mean to to do that, thanks. I 'll fix it